fdbserver/ptxn/test/FakeSequencer.actor.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * FakeSequencer.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbserver/ptxn/test/FakeSequencer.actor.h"
+
+#include <vector>
+
+#include "fdbserver/ptxn/test/Utils.h"
+
+#include "flow/actorcompiler.h" // must be the last file being included
+
+namespace ptxn::test {
+
+namespace {
+
+Future<Void> getVersion(std::shared_ptr<FakeSequencerContext> pContext, GetCommitVersionRequest request) {
```

Comment:
This is not an actor, so you can just return `void`.
```suggestion
void getVersion(std::shared_ptr<FakeSequencerContext> pContext, GetCommitVersionRequest request) {
```